app/models/invitation.rb

```diff
@@ -0,0 +1,10 @@
+class Invitation < ActiveRecord::Base
+  include Tokenable
+
+  belongs_to :inviter, class_name: "User"
+
+  validates_presence_of :recipient_email, :recipient_name, :message
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.